.changes/unreleased/polaris-TINY-11560-2025-04-01.yaml

```
@@ -0,0 +1,6 @@
+project: polaris
+kind: Fixed
+body: Spellchecking would sometimes produce overlapping areas
```

We don't publish this library anymore, and I think this was a bug in a premium feature, so the changelog belongs there.

---

But it is still something we should put a changelog for, isn't it? Or is there a different reason it's in the changie-log?
As for the premium, I'll make that changelog ( and the already-tested tests ) once this is in, since otherwise it'll just fail the new tests ( since it needs this fix )

---

Fair point. If it's just a log for polaris, then, it's not a "spellchecking" fix
```suggestion
body: `Search.findmany()` would sometimes produce overlapping areas
```